pkgs/by-name/ju/justbuild/package.nix

```diff
@@ -170,7 +172,7 @@ stdenv.mkDerivation rec {
   '';
 
   meta = with lib; {
-    broken = true; # last successful build 2024-01-26
+    broken = stdenv.isDarwin;
```

Comment:
```suggestion
    platforms = platforms.linux;
```

Reply:
It was previously build-able on darwin, the upstream package is build-able on darwin, but this packaging of it is not.

<https://nixos.org/manual/nixpkgs/unstable/#sec-allow-unsupported-system>
> If a program ought to work on a certain platform, but doesn’t, the platform should be included in meta.platforms, but marked as broken